server/docs/source/index.html.md

```diff
@@ -582,6 +582,16 @@ to | The end date and time (example: `?to=2017-01-01T13:15:00.00Z`) | now
   "version": "0.1.0",
   "registry": "https://stack-registry.kontena.io",
   "expose": "peer",
+  "parent": {
+    "name": "parent-stack-name",
+    "id": "my-grid/parent-stack-name"
```

Comment:
The `id` can also be `null` if it's an orphaned child stack, and the parent is missing. It will still have the `name` in that case.

This can easily happen as a result of failed `kontena stack install` commands, where the child stack is fine, but the parent stack fails to validate/install.